File: Python/uri_1045.py
a, b, c = map(float, input().split(' '))
v = [a, b, c]
v.sort(reverse=True)
a, b, c = v
if a >= (b + c):
    print('NAO FORMA TRIANGULO')
else:
    if a ** 2 == (b ** 2 + c ** 2):
        print('TRIANGULO RETANGULO')
    if a ** 2 > b ** 2 + c ** 2:
        print('TRIANGULO OBTUSANGULO')
    if a ** 2 < b ** 2 + c ** 2:
        print('TRIANGULO ACUTANGULO')
    if a == b == c:
        print('TRIANGULO EQUILATERO')
    elif a == b or c == a or b == c:
        print('TRIANGULO ISOSCELES')

File: Go/uri_1045.go
package main

import (
	"fmt"
	"math"
	"sort"
)

func main() {
	var a, b, c float64
	fmt.Scan(&a, &b, &c)
	v := [] float64{a, b, c}
	sort.Float64s(v)
	for i := 0; i < 3; i++ {
		v[i] = v[i]
	}
	// A = v[2] B = v[1] C[0]
	if v[2] >= v[1]+v[0] {
		fmt.Print("NAO FORMA TRIANGULO\n")
	} else {
		if math.Pow(v[2], 2) == (math.Pow(v[1], 2) + math.Pow(v[0], 2)) {
			fmt.Print("TRIANGULO RETANGULO\n")
		}
		if math.Pow(v[2], 2) > math.Pow(v[1], 2)+math.Pow(v[0], 2) {
			fmt.Print("TRIANGULO OBTUSANGULO\n")
		}
		if math.Pow(v[2], 2) < math.Pow(v[1], 2)+math.Pow(v[0], 2) {
			fmt.Print("TRIANGULO ACUTANGULO\n")
		}
		if v[2] == v[1] && v[1] == v[0] && v[0] == v[2] {
			fmt.Print("TRIANGULO EQUILATERO\n")
		} else if v[2] == v[1] || v[1] == v[0] || v[2] == v[0] {
			fmt.Print("TRIANGULO ISOSCELES\n")
		}
	}
}
